### ISSUE TITLE
Interactive window should be more aligned with the accessibility experience of VS Code 

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#8880]
Testing -161850

Since the interactive window is using a lot of custom UI this means that our accessibility users are maybe not used to the interaction it offers.
I tested the improved keyboard navigation in the test plan and it works as expected, but I think we need to take a step back and look at the bigger picture of the interactive window.

For example:
* Can we learn from the Debug Console and Terminal and have similar keyboard interaction as they do since this is something that our users are already familiar with
* Can we use the same navigation as we do in other VS Code toolbars? It is only one tab stop, and left/right are used to navigate through actions
* How do we move focus to the top level toolbar? Does that top level toolbar behave like other toolbars in vscode? Does it have the required aria roles?
* etc...
